ts/screens/IngressScreen.tsx

```diff
@@ -41,9 +45,27 @@ const styles = StyleSheet.create({
  */
 class IngressScreen extends React.PureComponent<Props> {
   public componentDidMount() {
-    // Dispatch START_APPLICATION_INITIALIZATION to initialize the app
-    this.props.dispatch(startApplicationInitialization());
-
+    // if device is rooted and it isn't an emulator
+    if (JailMonkey.isJailBroken() && !DeviceInfo.isEmulator()) {
+      Alert.alert(
+        I18n.t("startup.rootedDeviceAlert.msgTitle"),
+        I18n.t("startup.rootedDeviceAlert.msgText"),
+        [
+          {
+            text: I18n.t("global.buttons.ok"),
+            style: "destructive",
+            onPress: () => {
+              // after confirmation close app
+              RNExitApp.exitApp();
```

Comment:
Is a library needed to exit the application?
What do you think @cloudify? [https://facebook.github.io/react-native/docs/backhandler#exitapp](url)

Reply:
@Undermaken BackHandler only works for for Android (end partially for TVOS).